04_naive_bayes/04_naive_bayes.jl

```diff
@@ -262,40 +261,26 @@ As you can see below, the model (at least under this simple metric) is performin
 # ╔═╡ aa9f7ea4-2850-11eb-33e2-ade40fd0a360
 spam_filter_accurracy(x_test, y_test, spam_filter, 1)
 
-# ╔═╡ 55955e22-28fc-11eb-0153-83f34561665b
-# to create the emails corpus based on the frequencies of each word
-
-function create_spam_and_ham_corpus(model::BayesSpamFilter, df)
-	
-	n_spam_mails = sum(df.Prediction)
-	n_ham_mails = size(df)[1] - n_spam_mails
-	
-	ham_corpus = Array{String}(undef, n_ham_mails)
-	spam_corpus = Array{String}(undef, n_spam_mails)
-	
-	for i in 1:n_spam_mails
-		spam_corpus[i] = string([repeat(string(word, " "), 												model.words_count_spam[word]) for word in vocabulary])
-	end
-	
-	for i in 1:n_ham_mails
-		ham_corpus[i] = string([repeat(string(word, " "), 
-					model.words_count_ham[word]) for word in vocabulary])
-	end
-	return ham_corpus, spam_corpus
-end	
-
-# ╔═╡ e8ea2bd0-2903-11eb-176f-a527049a3968
-# ham_corpus, spam_corpus = create_spam_and_ham_corpus(spam_filter, raw_df)
-
-# ╔═╡ 7fb08fa2-29b9-11eb-39d5-f74320ac9609
+# ╔═╡ 3ce2070a-7156-11eb-1204-3d1850c7abee
 md"
-#### To do:
-* WordClouds
-* Confusion matrix
+### Summary
+In this chapter, we have used a naive-bayes approach to build a simple email spam filter. 
+First, a contextualization of the topic was made and the origin of the spam word was discussed.
+Then, the dataset and the theoretical framework was introduced. Using Bayes' theorem, and the data available, we assign a probability of belonging to a spam or a ham email to each word of the dataset. 
+Thus, the probability of a new email belonging to the spam category, is the product of the probabilities of each of its constituent words.
+Later, the data was pre-processed and a struct was defined for the spam filter object. Functions were implemented to fit the spam filter object to the data.
+Finally, a metric for evaluating the accuracy of the model was implemented, giving a result of approximately $0.95$.
 "
 
-# ╔═╡ 23fc8684-29d2-11eb-235c-29e2fad3bd5d
-corpus = "la la la le li li holaaa hola holu"
+# ╔═╡ 96834844-6d45-11eb-39a5-737dd8e43cb1
+md" ### References
+* [How the word 'Spam' came to mean 'Junk Message'](http://www.todayifoundout.com/index.php/2010/09/how-the-word-spam-came-to-mean-junk-message/)
+* [Monty Python Sketch - YouTube](https://www.youtube.com/watch?v=zLih-WQwBSc)
+* [What is Spam Filtering?](https://www.mailchannels.com/what-is-spam-filtering/)
+* [Naive Bayes Classifiers - GeeksforGeeks](https://www.geeksforgeeks.org/naive-bayes-classifiers/)
```

Comment:
GeeksforGeeks is like Wikipedia, try to find sources in books or authoritative sources. 
Also, let's not make our first two references about the origin of the word Spam, that should be the least important thing in this chapter. Move it to the end.